File: lab1.c
//c program to run the average of 3 test scores
#include <stdio.h>
#define NUM_TESTS 3

void main() {
    int test1, test2, test3, avgScore;
    printf("Welcome to Gradebot 1.0\n");
    printf("first test score: ");
    scanf("%d",&test1 );
    printf("\nsecond test score: ");
    scanf("%d",&test2 );
    printf("\nthird test score: ");
    scanf("%d",&test3 );
    
    avgScore = (test1 + test2 + test3) / NUM_TESTS;
    
    printf("Average Score: %d", avgScore);
}

File: lab1.js
let avgScore
let test1, test2, test3
const num_tests = 3
const pi = Math.PI

test1 = Number(prompt("Enter the score on the first test: "))
test2 = Number(prompt("Enter the score on the second test: "))
test3 = Number(prompt("Enter the score on the third test: "))

avgScore = ((test1 + test2 + test3) / num_tests)
alert('Your Average Score: ' + avgScore)
